src/backend/joanie/urls.py

```diff
@@ -97,15 +96,26 @@
     ),
     path(
         f"api/{API_VERSION}/",
-        include([*router.urls, *lms_urlpatterns, *payment_urlpatterns]),
-    ),
-    path(
-        f"api/{API_VERSION}/courses/<uuid:course_id>/",
-        include(course_related_router.urls),
-    ),
-    path(
-        f"api/{API_VERSION}/organizations/<uuid:organization_id>/",
-        include(organization_related_router.urls),
+        include(
+            [
+                path(
+                    "courses/products/",
+                    api_client.CourseProductRelationViewSet.as_view({"get": "list"}),
+                    name="course_product_relations_list_all",
+                ),
```

Comment:
I didn't understand why you still need this endpoint?

Reply:
Because we need to be able to list relations for which the user has course accesses.

Reply:
So to resume we will have
- `/courses/<course_id>/products/` to list all course's product relations
- `/courses/<course_id>/products/<product_id>` to get a course product relation
- `/course-product-relations/` to list all course product relations
- `/course-product-relations/<relation_id>` to get a course product relations

Right ?

Reply:
I updated api and related tests according to our discussion.